src/test/java/sample/ApplicationTest.java

```diff
@@ -0,0 +1,16 @@
+package sample;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(classes = Application.class)
```

Comment:
Just to keep down on technical debt from the start, let's please remove `(classes = Application.class)` since it's unnecessary.